app/controllers/epp/domains_controller.rb

```diff
@@ -4,6 +4,7 @@ class DomainsController < BaseController
     before_action :find_domain, only: %i[info renew update transfer delete]
     before_action :find_password, only: %i[info update transfer delete]
     before_action :set_paper_trail_whodunnit
+    before_action :parsing_schemas_prefix_and_version
```

Comment:
Method name should be without `-ing` suffix, so just `parse_schema_prefix_and_version` will do.